front-end/src/components/UserForm/SignIn.tsx

```diff
@@ -7,26 +7,53 @@ interface UserForm {
 interface State {
     isSubmitButtonEnable: boolean,
     username: string,
+    showBtnInPasswordInput: boolean,
+    isShowingPassword: boolean,
 }
 
 class SignIn extends React.Component<UserForm, State>{
 
     state = {
         isSubmitButtonEnable: false,
-        username: ""
+        username: "",
+        showBtnInPasswordInput: false,
+        isShowingPassword: false
     }
 
-    handleUserNameChange = (e: { target: { value: string; }; }) => {
+    handleUserNameChange = (e: { target: { value: string; }; }): void => {
```

Comment:
Lets keep consistency in the way we name event, we can use "e" or "event" but not both